newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/MaxProjectCountAlertMessage.js

```diff
@@ -7,21 +7,41 @@ import RaisedButton from '../../../../UI/RaisedButton';
 import AlertMessage from '../../../../UI/AlertMessage';
 import { Line, Column } from '../../../../UI/Grid';
 import { type Limits } from '../../../../Utils/GDevelopServices/Usage';
+import { type AuthenticatedUser } from '../../../../Profile/AuthenticatedUserContext';
+import Window from '../../../../Utils/Window';
 
 type Props = {|
   onUpgrade: () => void,
   limits: Limits,
 |};
 
+// Search "activate cloud projects" in the codebase for everything to
+// remove once cloud projects are activated for the desktop app.
+const supportsCloudProjects = Window.isDev();
```

Comment:
same here, could this be removed?

Reply:
(resolving to have just one conversation about this)